code/__HELPERS/unsorted.dm

```diff
@@ -499,9 +499,7 @@ Turf and target are seperate in case you want to teleport some distance from a t
 		var/atom/A = processing_list[1]
 		processing_list -= A
 
-		for(var/atom/a in A)
-			if(!(a in assembled))
-				processing_list |= a
+		processing_list |= (A.contents - assembled)
```

Comment:
i didn't test it, so it may not actually work, a.contents is a special list. generally when doing these operations on special lists, it just makes a copy, but somebody should download this and test to make sure byond doesn't just throw a stupid error message instead